### PR TITLE
RuleViolationResolution: Add `cant_fix` as reason

### PR DESCRIPTION
**Pull request by fviernau**
Signed-off-by: Frank Viernau <frank.viernau@here.com>

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1225)

### CODE REVIEW

File: model/src/main/kotlin/config/RuleViolationResolutionReason.kt [hunk 1]
```diff
@@ -20,6 +20,11 @@ package com.here.ort.model.config
  */
 
 enum class RuleViolationResolutionReason {
+    /**
+     * The rule violation cannot be fixed an is acceptable in this case.
```
Comment (sschuberth) [inline model/src/main/kotlin/config/RuleViolationResolutionReason.kt]: s/an/and/

File: model/src/main/kotlin/config/RuleViolationResolutionReason.kt [hunk 2]
```diff
@@ -20,6 +20,11 @@ package com.here.ort.model.config
...
+     */
+    CANT_FIX_EXCEPTION,
```
Comment (sschuberth) [inline model/src/main/kotlin/config/RuleViolationResolutionReason.kt]: I'd prefer to more formal `CANNOT_FIX_EXCEPTION`.

Comment (fviernau) [reply model/src/main/kotlin/config/RuleViolationResolutionReason.kt]: I'd be ok with that. Are you aware of the `ErrorResolutionReason.CANT_FIX_ISSUE` ?

Comment (sschuberth) [reply model/src/main/kotlin/config/RuleViolationResolutionReason.kt]: Damn. Then let's stick to `CANT` for consistency.

**Comment by fviernau** [comment]
merging this even if the appveyor fails, as that failure is not related to this PR and addressed by https://github.com/heremaps/oss-review-toolkit/pull/1228 .

**Pull request merged**